cpp/examples/lct_secir.cpp

```diff
@@ -0,0 +1,91 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Lena Ploetzke
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "lct_secir/model.h"
+#include "lct_secir/infection_state.h"
+#include "lct_secir/simulation.h"
+#include "memilio/config.h"
+#include "memilio/utils/time_series.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+#include "memilio/math/eigen.h"
+#include <vector>
+
+int main()
+{
+    /** Simple example to demonstrate how to run a simulation using an LCT SECIR model. 
+    Parameters, initial values and subcompartments are not meant to represent a realistic scenario. */
+
+    // Set vector that specifies the number of subcompartments.
+    std::vector<int> num_subcompartments((int)mio::lsecir::InfectionStateBase::Count, 1);
+    num_subcompartments[(int)mio::lsecir::InfectionStateBase::Exposed]            = 2;
+    num_subcompartments[(int)mio::lsecir::InfectionStateBase::InfectedNoSymptoms] = 3;
+    num_subcompartments[(int)mio::lsecir::InfectionStateBase::InfectedCritical]   = 5;
+    mio::lsecir::InfectionState infectionState(num_subcompartments);
```

Comment:
The notation infectionState is inconsistent with the remaining notation, maybe use infection_state?

Reply:
What about this?

Reply:
Forgot to add the file to the commit... thanks! :)